utils/coreutils/utils.go

```diff
@@ -400,3 +400,17 @@ func SetCliExecutableName(executableName string) {
 func GetCliExecutableName() string {
 	return cliExecutableName
 }
+
+// Get the explicit list by number of elements.
+// For example for multiple elements: "one, two and three".
```

Comment:
For example for multiple elements: "one, two and three".
-->
For example, turn ["one","two", "three"] into "one, two and three".